tools/serverpod_cli/lib/src/generator/dart/library_generators/library_generator.dart

```diff
@@ -655,6 +656,15 @@ class LibraryGenerator {
     return '$experimentalWarning\n///\n$documentationComment';
   }
 
+  Iterable<Expression> _buildEndpointCallAnnotations(
+      MethodDefinition methodDef) {
+    return methodDef.annotations.map((annotation) => refer(annotation
+                .arguments !=
+            null
+        ? '${annotation.name}(${annotation.arguments!.map((a) => "'$a'").join(',')})'
```

Comment:
Suggestion: Remove dangerous deref `!`.

If you expand the lambda I think you should be able to use an early return in combination with investigating the value to avoid a dangerous deref.